templates/web/base/admin/body.html

```diff
@@ -142,4 +142,6 @@
 </div>
 [% END %][%# Only show all the above if no errors with category form %]
 
+[% INCLUDE 'admin/_translations.html' action=c.uri_for('body', body_id) %]
```

Comment:
Should this be contained within the edit body details box? It sort of feels like it's part of editing the body details after all. But I realise that it's a separate form. Should existing translations be editable in the one form along with the other body details? And put up with the main "name" field?